packages/features/embed/Embed.tsx

```diff
@@ -1944,22 +1090,34 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
   );
 };
 
-export const EmbedDialog = () => {
+export const EmbedDialog = ({
+  types,
+  tabs,
+  eventTypeHideOptionDisabled,
+}: {
+  types: EmbedTypes;
+  tabs: EmbedTabs;
+  eventTypeHideOptionDisabled: boolean;
+}) => {
   const router = useRouter();
   const embedUrl: string = router.query.embedUrl as string;
   return (
     <Dialog name="embed" clearQueryParamsOnClose={queryParamsForDialog}>
       {!router.query.embedType ? (
-        <ChooseEmbedTypesDialogContent />
+        <ChooseEmbedTypesDialogContent types={types} />
```

Comment:
It would allow types to be filtered outside the EmbedDialog component. It now doesn't need to worry if it's routing-form or event-type. Thanks again @zomars @sean-brydon 🙏